resources/js/Directives/getinvolvedDirective.js

```diff
@@ -8,16 +8,17 @@ app.directive('getinvolved', ['$http', function ($http) {
             self.contributors
 
             function excludeBot(contributors) {
-                return contributors.filter(function(member){
+                return contributors.filter(function(member) {
```

Comment:
please do not modify lines unrelated to your assigned issue.

Reply:
okay..  you might check the PR now